apps/video/test/unit/video_test.js

```diff
@@ -1787,22 +1857,29 @@ suite('Video App Unit Tests', function() {
     test('#updateSelection: toggle thumbnail as selected', function() {
       var thumbnail = thumbnailList.thumbnailMap[videodata.name];
       thumbnail.htmlNode.classList.remove('selected'); // not selected
+      thumbnail.htmlNode.setAttribute('aria-selected', false);
       updateSelection(videodata);
       assert.isTrue(thumbnail.htmlNode.classList.contains('selected'),
                     'thumbnail should contain \'selected\' class');
+      assert.equal(thumbnail.htmlNode.getAttribute('aria-selected'), 'true',
+                   'thumbnail aria-selected attribute should be set to true');
     });
 
     test('#updateSelection: toggle thumbnail as not selected', function() {
       var thumbnail = thumbnailList.thumbnailMap[videodata.name];
       thumbnail.htmlNode.classList.add('selected'); // selected
+      thumbnail.htmlNode.setAttribute('aria-selected', true);
       updateSelection(videodata);
       assert.isFalse(thumbnail.htmlNode.classList.contains('selected'),
                      'thumbnail should not contain \'selected\' class');
+      assert.equal(thumbnail.htmlNode.getAttribute('aria-selected'), 'false',
+                   'thumbnail aria-selected attribute should be set to false');
     });
 
     test('#updateSelection: update (add to) selected filenames', function() {
```

Comment:
Looking at these tests, I don't see the difference between this test and the first 'updateSelection' test. They both seem to be testing the same scenario; they assert different information for some reason. Do you see any difference in these tests. If not, can you combine this with the first test so that there is one test with a union of the asserts from the two tests? That would be great.